src/components/LoanCards/LoanTags/LoanTag.vue

```diff
@@ -39,6 +39,13 @@ export default {
 			default: 'endingSoon'
 		}
 	},
+	mounted() {
+		this.$kvTrackEvent(
+			'loan-card',
+			'view-tag',
```

Comment:
'view-tag' is not in the web analytics action list. Can you provide a screenshot or link to this feature so I can suggest analytics for this? Not sure where these tags are

Reply:
They look like these https://github.com/kiva/ui/pull/4336

Reply:
can change to `'loan-card', 'show', 'view-loan-tag', this.variation`

Reply:
Let's go with: ['loan-card', 'show', 'tag-almost-funded']

Reply:
sgtm ty!!